dex/pair/README.md

```diff
@@ -213,3 +213,173 @@ The interaction scripts are located in the _dex/interaction_ directory. The scri
+
+### getLpTokensSafePriceByRoundOffset
+
+```rust
+    #[view(getLpTokensSafePriceByRoundOffset)]
+    fn get_lp_tokens_safe_price_by_round_offset(
+        &self,
+        pair_address: ManagedAddress,
+        round_offset: Round,
+        liquidity: BigUint,
+    ) -> MultiValue2<EsdtTokenPayment, EsdtTokenPayment>
+```
+
+This function allows you to simulate the value of both tokens within a liquidity pool based on a provided LP token amount. It receives the pair address, a round_offset, and the amount of LP tokens as parameters. It works in the same way as the __getSafePriceByRoundOffset__ function for the round_offset, by computing the end round and the start round automatically, using the current block round and the round offset variable.
+The function returns two output payments, one for each token in the pair, with their values computed at the safe price.
+
+### getLpTokensSafePriceByDefaultOffset
+
+```rust
+    #[view(getLpTokensSafePriceByDefaultOffset)]
+    fn get_lp_tokens_safe_price_by_default_offset(
+        &self,
+        pair_address: ManagedAddress,
+        liquidity: BigUint,
+    ) -> MultiValue2<EsdtTokenPayment, EsdtTokenPayment>
+```
+
+This function allows you to simulate the value of both tokens within a liquidity pool based on a provided LP token amount. It receives the pair address and the amount of LP tokens as parameters and works similarly to __getLpTokensSafePriceByRoundOffset__ endpoint, but instead of using a provided round_offset variable, it uses a default one which at this moment is set to 600 rounds (which translates in an one hour window).
+The function returns two output payments, one for each token in the pair, with their values computed at the safe price.
+
+### getLpTokensSafePriceByTimestampOffset
+
+```rust
+    #[view(getLpTokensSafePriceByTimestampOffset)]
+    fn get_lp_tokens_safe_price_by_timestamp_offset(
+        &self,
+        pair_address: ManagedAddress,
+        timestamp_offset: u64,
+        liquidity: BigUint,
+    ) -> MultiValue2<EsdtTokenPayment, EsdtTokenPayment>
+```
+
+This function allows you to simulate the value of both tokens within a liquidity pool based on a provided LP token amount. It receives the pair address, a timestamp_offset and the amount of LP tokens as parameters. Again, like the __getLpTokensSafePriceByRoundOffset__ endpoint, which automatically computes the start and end rounds of the query, this function calculates the round offset by dividing the timestamp_offset to a generic __SECONDS_PER_ROUND__ constant value.
+The function returns two output payments, one for each token in the pair, with their values computed at the safe price.
+
+## Legacy endpoints
+
+In order to avoid backwards compatibility issues, the two legacy endpoints from Safe Price V1 were kept, but they now use the new Safe Price V2 logic. One important aspect here is that they are not part of the Safe Price V2 view factory contract, but instead they are actual endpoints in the __Pair SC__.
```

Comment:
```suggestion
In order to avoid backwards compatibility issues, the two legacy endpoints from Safe Price V1 were kept, but they now use the new Safe Price V2 logic. One important aspect here is that they are not part of the Safe Price V2 view factory contract, but instead they are actual endpoints in the __Pair SC__.
__Important. These endpoints are planned to be phased out at any point in future, thus we consider them as deprecated and we recommend the usage of the ones available in the View factory contract.__
```